src/components/DailyStatistics/DailyStatisticsScreen.tsx

```diff
@@ -0,0 +1,173 @@
+import { StatisticsContext } from "../../context/statistics";
+import { getDailyStatistics } from "../../services/statistics";
+import { summariseTransactions } from "./utils";
+import { TransactionHistoryCard } from "./TransactionHistoryCard";
+import { StatisticsHeader } from "./StatisticsHeader";
+import { addDays, subDays, getTime } from "date-fns";
+
+const styles = StyleSheet.create({
+  content: {
+    padding: size(2),
+    paddingVertical: size(6),
+    height: "100%",
+    width: 512,
+    maxWidth: "100%"
+  },
+  headerText: {
+    marginBottom: size(4)
+  },
+  bannerWrapper: {
+    marginBottom: size(1.5)
+  },
+  categoryName: {
+    fontFamily: "brand-bold",
+    fontSize: fontSize(1),
+    marginBottom: size(3)
+  }
+});
+
+const DailyStatisticsScreen: FunctionComponent<NavigationFocusInjectedProps> = ({
+  navigation,
+  isFocused
+}) => {
+  useEffect(() => {
+    Sentry.addBreadcrumb({
+      category: "navigation",
+      message: "DailyStatisticsScreen"
+    });
+  }, []);
+
+  const messageContent = useContext(ImportantMessageContentContext);
+  const { config } = useConfigContext();
+  const showHelpModal = useContext(HelpModalContext);
+  const checkUpdates = useCheckUpdates();
+  const { sessionToken, endpoint, operatorToken } = useContext(AuthContext);
+  const { totalCount, setTotalCount } = useContext(StatisticsContext);
+  const {
+    currentTimestamp,
+    setCurrentTimestamp,
+    lastTransactionTime,
+    setLastTransactionTime,
+    transactionHistory,
+    setTransactionHistory
+  } = useContext(StatisticsContext);
+
+  const { policies } = useContext(CampaignConfigContext);
+
+  const fetchDailyStatistics = async (
+    currentTimestamp: number
+  ): Promise<void> => {
+    const response = await getDailyStatistics(
+      currentTimestamp,
+      sessionToken,
+      endpoint,
+      [operatorToken]
+    );
+
+    const {
+      summarisedTransactionHistory,
+      summarisedTotalCount
+    } = summariseTransactions(response, policies);
+
+    setTransactionHistory(summarisedTransactionHistory);
+    setTotalCount(summarisedTotalCount);
+    setCurrentTimestamp(currentTimestamp);
+
+    if (response.pastTransactions.length !== 0) {
+      setLastTransactionTime(response.pastTransactions[0].transactionTime);
+    } else {
+      setLastTransactionTime(0);
+    }
+  };
+
+  const onPressPrevDay = (): void => {
+    const prevDay = getTime(subDays(currentTimestamp, 1));
+    setCurrentTimestamp(prevDay);
+    fetchDailyStatistics(prevDay);
+  };
+
+  const onPressNextDay = (): void => {
+    const nextDay = getTime(addDays(currentTimestamp, 1));
+    setCurrentTimestamp(nextDay);
+    fetchDailyStatistics(nextDay);
+  };
+
+  useEffect(() => {
+    // When entering the first time
+    if (totalCount === null) {
+      fetchDailyStatistics(Date.now());
+    }
+  });
+  useEffect(() => {
+    if (isFocused) {
+      checkUpdates();
+    }
+  }, [isFocused, checkUpdates]);
```

Comment:
If this is used to check whether to update the whole application, maybe this can be removed here?